src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -783,6 +783,19 @@ public function testFloatKeys()
 
         $this->assertEquals($expected, $this->parser->parse($yaml));
     }
+
+    /**
+     * @group legacy
+     * throw ParseException in Symfony 3.0
+     */
+    public function testColonInMappingValueException()
```

Comment:
I've done that in Twig: see https://github.com/twigphp/Twig/pull/1897/files#diff-cac3b27cf2f7ec6cf8fdb075ac3eccfbR277